app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -84,9 +84,14 @@
       </div>
       <div class="menu">
         {{#each speakers as |speaker|}}
-          <div data-value="{{map-value mapper speaker}}" class="item">
-            {{speaker.name}}
-          </div>
+          {{#if noSpeakerExists}}
+            <div data-value="null">
+            </div>
+          {{else}}
+            <div data-value="{{map-value mapper speaker}}" class="item">
```

Comment:
Just wrap this in 
```hbs
{{#if speaker.name}}

```
This does the trick too.

Reply:
is the changes I make is causing any error? @uds5501

Reply:
@uds5501 cal

> Just wrap this in
> 
> ```
> {{#if speaker.name}}
> ```
> 
> This does the trick too.

Can you clarify this??@uds5501